packages/sushi/src/contract-addresses.ts

```diff
@@ -0,0 +1,28 @@
+import type { Address } from 'viem'
+import { Chains } from './utils'
+
+export const CHAIN_TO_CONTRACTS: { [_chainId: number]: Address[] | undefined } =
+  {
+    [Chains.ETHEREUM]: [
+      '0x5550D13389bB70F45fCeF58f19f6b6e87F6e747d', // RouteProcessor3_2
+      '0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F', // V2Router02
+    ],
+    [Chains.OPTIMISM]: [
+      '0xEb94EcA012eC0bbB254722FdDa2CE7475875A52B', // RouteProcessor3_2
```

Comment:
No `V2Router02` on OP?

Reply:
Not that I could find. It is not listed in the docs or on their SDK.